reactioncommerce/reaction
# Remove storefront catalog grid

Impact: **minor**  
Type: **feature**

## Changes
- When you are not logged in or you are a customer, you no longer see a catalog grid on the home page or tag pages
- When you are not logged in, you no longer see search or tags in the nav bar
- When you are logged in as an admin, there is no longer an "Edit mode" toggle in the nav bar. The app is always in edit mode.

## Breaking changes
These are all breaking changes in terms of using the Meteor app as a storefront, but it is no longer intended to be used as a storefront in the 2.0 release.

## Testing
- Verify the changes listed above.
- Verify that a typical product editing flow continues to work as an admin
- Verify that the new operator route is unaffected
